scripts/globals/items/sheep_cap_+1.lua

```diff
@@ -0,0 +1,29 @@
+-----------------------------------------
+--  ID: 26720
+--  Sheep Cap +1
+--  When used, you will obtain one of eleven random items. They are:
+--  Faerie Apple, Sunflower Seeds, Red Moko Grass, La Theine Cabbage, Boyahda Moss,
+--  Pine Nuts, Beaugreens, Acorn, Dung, Batagreens, and Moko Grass
+-----------------------------------------
+
+-----------------------------------------
+-- OnItemCheck
+-----------------------------------------
+
+function onItemCheck(target)
+    local result = 0;
+    if (target:getFreeSlotsCount() == 0) then
+        result = 308;
+    end
+    return result;
+end;
+
+-----------------------------------------
+-- OnItemUse
+-----------------------------------------
+
+function onItemUse(target)
+    local giftList = {4363,4505,1845,4366,919,2213,4571,4504,8929,4367,833}
+    local gift = math.random(0,10);

```

Comment:
Ok I confirmed with a print that that giftList[0] didn't print, and nothing after it happened. But that leaves me confused as hell as to why my Esuna script is able to do anything when there's only one removable status.